src/Uno.UI/UI/LayoutHelper.cs

```diff
@@ -375,7 +375,6 @@ internal static Size AtLeastZero(this Size value)
 			}
 		}
 
-		[Pure]
```

Comment:
?

Reply:
This method is definitely not pure...

Reply:
How so? It modifies `rect1`, but Rect is a struct so it's modifying a copy of the value that was passed to the method. It should have no side-effects from the point of view of the caller.

Am I missing something? Is my mental idea of 'Pure' not pure enough?

Reply:
You're right.